Content.Server/Body/Components/MetabolizerComponent.cs

```diff
@@ -55,12 +55,12 @@ public sealed partial class MetabolizerComponent : Component
         public bool RemoveEmpty = false;
 
         /// <summary>
-        ///     How many reagents can this metabolizer process at once?
+        ///     How many poisons can this metabolizer process at once?
         ///     Used to nerf 'stacked poisons' where having 5+ different poisons in a syringe, even at low
         ///     quantity, would be muuuuch better than just one poison acting.
         /// </summary>
         [DataField("maxReagents")]
-        public int MaxReagentsProcessable = 3;
+        public int MaxPoisonsProcessable = 3;
```

Comment:
I'm not sure if I understand the rationale behind the metabolism change.

As-is, chemicals will eventually metabolize if there are more than can be metabolized at once in the solution - is this to prevent buildup beyond stomach capacity to prevent cryo chem waste with the per-reagent injection?

If anything, species-specific care might be more nuanced with the metabolism as it was (oh, don't put too much in at once, the human/moth might not metabolize it all), and this disincentivizes the 5-in-1 cryo panacea cocktail.

Reply:
honestly i didn't even want to leave the poison limit, however because people are sweaty power gamers which required this to be added in the first place as to not do 20+ damage per second, i have left the poison restriction behind

Reply:
Do not change an upstream value like this by name, this is a fucking nightmare to deal with on upstream updates